eval.py

```diff
@@ -1220,10 +1222,11 @@ def evaluate(net:Yolact, dataset, train_mode=False):
             w_batch = []
 
         for it, image_idx in enumerate(itertools.cycle(dataset_indices)):
+            iteration = it // args.batch_size
             timer.reset()
-            if not args.num_iterations and it == args.warm_up_iterations + dataset_size:
+            if not args.num_iterations and iteration == args.warm_up_iterations + dataset_size:
                 break
-            elif args.num_iterations and it // args.batch_size > args.warm_up_iterations + args.num_iterations:
+            elif args.num_iterations and iteration > args.warm_up_iterations + args.num_iterations:
```

Comment:
fix break condition, iterations are zero indexed
```suggestion
            elif args.num_iterations and iteration >= args.warm_up_iterations + args.num_iterations:
```